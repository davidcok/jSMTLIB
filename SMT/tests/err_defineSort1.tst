; defining a sort before set-logic
(define-sort A () Bool)
