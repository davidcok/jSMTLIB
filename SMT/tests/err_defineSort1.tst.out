(define-sort A () Bool)
^^^^^^^^^^^^^^^^^^^^^^^
(error "The logic must be set before a define-sort command is issued")
